server/main.py

```diff
@@ -162,6 +162,8 @@ async def index(request):
             job.command('git checkout main')
             job.command(f'git merge-base --is-ancestor {quote(commit)} HEAD')
         job.command(f'git checkout {quote(commit)}')
+        job.command(f'git submodule init')
```

Comment:
Can we do a `--recurse-submodules` in the `clone` step instead?

Reply:
Don't think that would work if the submodule exists only in a branch, but not in main

Reply:
Only with a subsequent `git submodule update`

Reply:
Updated!